cdap-ui/app/cdap/services/resource-helper/index.js

```diff
@@ -33,7 +33,8 @@ function createApi (dataSrc, method, type, path, options = {}) {
       reqObj = Object.assign({}, reqObj, { headers });
     }
 
-    if (cookie.load('CDAP_Auth_Token')) {
+    // Since CDAP_AUTH_TOKEN won't have a JSON we need to parse we can skip parse.
+    if (cookie.load('CDAP_Auth_Token', false)) {
```

Comment:
I think this is actually wrong. This should check for `securityEnabled`. Then the JSON parse part can be added to line 39 and 40 (probably can just assign it to a variable first.